src/client/components/dashboard-page/dashboard-page.component.js

```diff
@@ -0,0 +1,150 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './dashboard-page.style.css';
+import SidebarMenu from '../side-navigation/sidebar.component';
+import ProgressBar from '../progress-bar/progress-bar.component';
+import Card from '../card/card.component';
+import { CardTitle } from '../card/card-title.component';
+import { CropSummary } from '../card/crop-summary-card/crop-summary-card.component';
+import LineChartForDashboard from '../line-chart-for-dashboard/line-chart-for-dashboard.component';
+import Footer from '../footer/footer.component';
+import Logout from '../logout/logout.component';
+
+export default function DashboardPage({
+  isVisible,
+  showDashboardFunc,
+  showBatchDetailsFunc,
+  showAddBatchFunc,
+  showLogoutModal,
+  userName,
+  logoutModal,
+  closeAction,
+  logoutFunc,
+  progressBarData,
+  lineChartData,
+}) {
+  return (
+    <div className="dashboard">
+      <SidebarMenu
+        isActive={false}
+        isVisible={isVisible}
+        showDashboard={showDashboardFunc}
+        showBatchDetails={showBatchDetailsFunc}
+        showAddBatch={showAddBatchFunc}
+        logout={showLogoutModal}
+      />
+      <Logout
+        userName={userName}
+        openState={logoutModal}
+        closeAction={closeAction}
+        logoutFunc={logoutFunc}
+      />
+      <div className="content">
+        <header>
+          <h1>Batch overview</h1>
+        </header>
+        <main>
+          <ProgressBar
+            startDate={progressBarData.startDate}
+            currentDate={progressBarData.currentDate}
+            stages={progressBarData.stages}
+          />
+          <div className="cards">
+            <CropSummary />
+            <Card>
+              <CardTitle title="Temperature" />
+              <LineChartForDashboard
+                tempData={lineChartData.tempData}
```

Comment:
But why not?
`<LineChartForDashboard lineChartData={lineChartData}/>`

Reply:
It's like your saying : "Look at this code!... and btw I'm not telling your what it is"